python/src/ops.cpp

```diff
@@ -3089,6 +3090,9 @@ void init_ops(py::module_& m) {
           w (array): Quantized matrix packed in unsigned integers
           scales (array): The scales to use per ``group_size`` elements of ``w``
           biases (array): The biases to use per ``group_size`` elements of ``w``
+          transpose (bool, optional): Defines whether to multiply with the
+            transposed ``w`` or not, namelye whether we are performing
+            ``x @ w.T`` or ``x @ w``. (default: True)
           group_size (int, optional): The size of the group in ``w`` that
             shares a scale and bias. (default: 64)
```

Comment:
Sorry very nitpicky, but I'm on a crusade to standardize double backticks on Default values (e.g. for `True` and `64`)